Lodgify.Payments.Stripe.Domain/Lodgify.Payments.Stripe.Domain.csproj

```
@@ -6,4 +6,14 @@
         <Nullable>enable</Nullable>
     </PropertyGroup>
 
+    <ItemGroup>
+        <SonarQubeSetting Include="sonar.coverage.exclusions">
+            <Value>**/AccountProperty.cs,**/AccountUpdatedEvent.cs</Value>
```

We do not have `AccountUpdatedEvent` anymore
```suggestion
            <Value>**/AccountProperty.cs</Value>
```